src/features/controls.ts

```diff
@@ -125,17 +126,20 @@ export const ControlsMixin = (ModelViewerElement:
           this[$promptElement] =
               this.shadowRoot!.querySelector('.controls-prompt')!;
 
-          this[$camera] = scene.getCamera();
-
-          this[$controls] = new SmoothControls(this[$camera], scene.canvas);
-          this[$controls].target.set(0, 0, 0);
+          this[$controls] = new SmoothControls(scene.getCamera(), scene.canvas);
+          this[$updateCameraOrbit]();
+          this[$updateCameraFov]();
         }
 
         getCameraOrbit(): SphericalPosition {
           const {theta, phi, radius} = this[$lastSpherical];
           return {theta, phi, radius};
         }
 
+        getCameraFov(): number {
```

Comment:
Let's change this to `getFieldOfView()` to be inline with the property/attribute